browser/css/jssidebar.css

```diff
@@ -241,17 +241,14 @@ div.sidebar.ui-grid .checkbutton.sidebar,
 #PosSizePropertyPanel.sidebar #ratio input[type='checkbox'] {
 	margin-inline-start: 0;
 }
+
+#ParaPropertyPanel.sidebar #indentfieldbox,
 #ParaPropertyPanel.sidebar #box3 {
 	display: grid;
 	justify-content: start;
 	row-gap: 4px;
 }
 
-#ParaPropertyPanel.sidebar #indentfieldbox {
-	display: grid;
-	row-gap: 4px;
-}
-
 #ParaPropertyPanel .sidebar.ui-grid-cell {
```

Comment:
Thanks! if you could have this part in a different commit or PR I could merge it right away

Reply:
see #6439